content/ja/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -207,4 +207,4 @@ Kubernetes1.7とそれ以降のバージョンにおいて、StatefulSetの`.spe
 
 * [ステートフルなアプリケーションのデプロイ](/ja/docs/tutorials/stateful-application/basic-stateful-set/)の例を参考にしてください。
 * [StatefulSetを使ったCassandraのデプロイ](/ja/docs/tutorials/stateful-application/cassandra/)の例を参考にしてください。
-* [レプリカを持つステートフルアプリケーションを実行する](/ja/docs/tasks/run-application/run-replicated-stateful-application/)の例を参考にしてください。
+* [レプリカを持つステートフルアプリケーションを実行する](/ja/docs/tasks/run-application/run-replicated-stateful-application/)の例を参考にしてください。
```

Comment:
This difference is due to the loss of a line break at the end of the file.
I think this change is not necessary.

Reply:
My editor removed the trailing newline automatically.

I've re-added it and amended the commit.